handout/components/app_structure/responding_to_component_events.md

```diff
@@ -6,42 +6,47 @@ An event handler is specified inside the template using round brackets to denote
 import {Component} from '@angular/core';
 
 @Component({
-  selector: 'counter',
+  selector: 'rio-counter',
   template: `
     <div>
-      <p>Count: {{ num }}</p>
+      <p>Count: {{num}}</p>
       <button (click)="increment()">Increment</button>
     </div>
   `
 })
-export class Counter {
-  num: number = 0;
+export class CounterComponent {
+  num: number;
+
+  constructor() {
+    this.num = 0;
+  }
 
   increment() {
     this.num++;
   }
 }
 ```
 
-[View Example](http://plnkr.co/edit/15wHrpea6GY7yLr7hl61?p=preview)
+[View Example](http://plnkr.co/edit/l4FweMxodN8I26OeqhGH?p=preview)
 
 To send data out of components via outputs, start by defining the outputs attribute. It accepts a list of output parameters that a component exposes to its parent.
 
+`app/counter.component.ts`
 ```js
-import {Component, EventEmitter, Input, Output} from '@angular/core';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
 
 @Component({
-  selector: 'counter',
-  template: `
-    <div>
-      <p>Count: {{ count }}</p>
-      <button (click)="increment()">Increment</button>
-    </div>
-  `
+  selector: 'rio-counter',
+  templateUrl: 'app/counter.component.html'
 })
-export class Counter {
-  @Input() count: number = 0;
-  @Output() result: EventEmitter = new EventEmitter();
```

Comment:
Why not use TypeScript's short cut?  Also shouldn't this be typed with a generic? `EventEmitter<number>`?